### PR TITLE
Trello-WZWk6mKa: checking if admin with query filters when updating m…

### PR DESCRIPTION
**Pull request by marcosraimondi1**
…anagement

- Check if is admin by using query filters to get project management to update (previously only querying mgmt by project id)
- Updated relevant tests
- Tested with postman

### CODE REVIEW

**Comment by marcosraimondi1** [comment]
There are previous failing tests, using mvn package -DskipTests

File: src/main/kotlin/com/angorasix/projects/management/core/infrastructure/persistence/repository/ProjectManagementFilterRepositoryImpl.kt [hunk 1]
```diff
@@ -20,10 +22,31 @@ class ProjectManagementFilterRepositoryImpl(val mongoOps: ReactiveMongoOperation
...
+    ids?.let { query.addCriteria(where("_id").`in`(it)) }
     projectIds?.let { query.addCriteria(where("projectId").`in`(it)) }
+
+    if (requestingOthers) {
+        return query
+    }
+
+    if (requestingOwn) {
+        query.addCriteria(where("admins").elemMatch(where("contributorId").`is`(simpleContributor?.contributorId)))
+    }
```
Comment (rozagerardo) [inline src/main/kotlin/com/angorasix/projects/management/core/infrastructure/persistence/repository/ProjectManagementFilterRepositoryImpl.kt:41]: If this comment makes sense:
https://github.com/AngoraSix/projects.presentations/pull/22#discussion_r1444159204
Then we probably want to simplify the logic similarly here :)

File: src/main/kotlin/com/angorasix/projects/management/core/presentation/handler/ProjectsManagementHandler.kt [hunk 2]
```diff
@@ -187,7 +187,11 @@ class ProjectsManagementHandler(
                 "Project Management",
             )
         }
-        return service.updateProjectManagement(projectId, updateProjectManagementData)?.let {
+        return service.updateProjectManagement(
```
Comment (rozagerardo) [inline src/main/kotlin/com/angorasix/projects/management/core/presentation/handler/ProjectsManagementHandler.kt:209]: Same comment as https://github.com/AngoraSix/projects.presentations/pull/22/files#r1444145942

**Comment by rozagerardo** [comment]
Cool, nice work brate

**Pull request merged**